src/test/kotlin/uk/gov/justice/digital/hmpps/createandvaryalicenceapi/integration/ComIntegrationTest.kt

```diff
@@ -66,7 +66,7 @@ class ComIntegrationTest : IntegrationTestBase() {
           "Staff Surname",
           "A01B02C",
           "Test Team",
```

Comment:
Now we are doing date comparisons when checking for past release dates, need to adjust this seed data.